integration/plugin/logging/main_test.go

```diff
@@ -19,6 +19,9 @@ func TestMain(m *testing.M) {
 		fmt.Println(err)
 		os.Exit(1)
 	}
+	if testEnv.OSType != "linux" {
+		os.Exit(0)
+	}
```

Comment:
We should probably consider skipping the tests on Windows, instead of this exit (see https://github.com/moby/moby/pull/40193)